packages/xchain-bitcoin/src/client.ts

```diff
@@ -311,9 +311,9 @@ class Client implements BitcoinClient, XChainClient {
     }
     const calcdFees: Fees = {
       type: feesOptions.type,
-      fast: baseAmount(feesOptions.fast!.amount().multipliedBy(nextBlockFeeRate)),
-      average: baseAmount(feesOptions.average!.amount().multipliedBy(nextBlockFeeRate)),
-      fastest: baseAmount(feesOptions.fastest!.amount().multipliedBy(nextBlockFeeRate)),
+      fast: baseAmount(feesOptions.fast?.amount().multipliedBy(nextBlockFeeRate)),
+      average: baseAmount(feesOptions.average?.amount().multipliedBy(nextBlockFeeRate)),
+      fastest: baseAmount(feesOptions.fastest?.amount().multipliedBy(nextBlockFeeRate)),
```

Comment:
@HelloKashif ^ It will break if `feesOptions.fast` or `feesOptions.average` or `feesOptions.fastest` is undefined.

Reply:
@veado feesOptions is defined above so it shouldn't be undefined.
We can either move to ! mark instead of ?
OR
put a default value using ?? like
`feesOption.fast?.amount() ?? 5`

which one do you recommend?

Reply:
I would recommend to change `type Fees` as follow to avoid optional values.
```
type Fees = {
    type: FeeType
    fastest: BaseAmount
    fast: BaseAmount
    average: BaseAmount
};
```
If there is any chain, which provides one fee only, we will set same value for `average`, `fast`, `fastest`. It will be easier for frontends, too.